### PR TITLE
openclosedriveeject: Add version 1.33

### PR DESCRIPTION
**Pull request by worldadventurer**
Used https://github.com/lukesampson/scoop-extras/blob/master/bucket/quicktextpaste.json as the base template.

### CODE REVIEW

File: bucket/openclosedriveeject.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            "hash": "fa09f13605c944efd606a40b1a267633bf3baa5a885c52fb52de75d3a81d15d8",
+            "bin": [
+                [
+                    "OpenCloseDriveEject_p.exe",
+                    "OpenCloseDriveEject"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "OpenCloseDriveEject_p.exe",
```
Comment (Ash258) [inline bucket/openclosedriveeject.json:37]:
```suggestion
                    "OpenCloseDriveEject.exe",
```

File: bucket/openclosedriveeject.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        },
+        "32bit": {
+            "url": "https://www.softwareok.com/Download/OpenCloseDriveEject.zip",
+            "hash": "fa09f13605c944efd606a40b1a267633bf3baa5a885c52fb52de75d3a81d15d8",
+            "bin": [
+                [
+                    "OpenCloseDriveEject_p.exe",
+                    "OpenCloseDriveEject"
+                ]
+            ],
```
Comment (Ash258) [inline bucket/openclosedriveeject.json:34]:
```suggestion
            "bin": "OpenCloseDriveEject.exe",
```

File: bucket/openclosedriveeject.json (hunk 1 above)
Comment (worldadventurer) [reply bucket/openclosedriveeject.json:37]: Thanks for catching the x32 difference in filenames!

File: bucket/openclosedriveeject.json (hunk 2 above)
Comment (worldadventurer) [reply bucket/openclosedriveeject.json:34]: Thanks for catching the x32 difference in filenames!

**Comment by worldadventurer** [comment]
I made the changes you suggested (thanks for catching that!). Please review and approve at your convenience. :)

**Pull request merged**